components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -299,6 +407,45 @@ export function CreateWorkspacePage() {
     );
 }
 
+function RememberOptions(params: {
+    disabled?: boolean;
+    checked: boolean;
+    workspaceContext?: WorkspaceContext | null;
+    onChange: (checked: boolean) => void;
+}) {
+    const { disabled, checked, workspaceContext, onChange } = params;
+
+    const label =
+        "Always use these options for " +
+        (CommitContext.is(workspaceContext)
+            ? workspaceContext.repository.owner + "/" + workspaceContext.repository.name
+            : "this repository") +
+        ".";
```

Comment:
**thought**: Repeating the repository here sounds a bit duplicate as it's already included in the first dropdown, no? The repository can also become more prominent, see relevant discussion (internal).